documentation/src/main/asciidoc/topics/server_protocols_hotrod.adoc

```diff
@@ -2,86 +2,27 @@
-cb.tcpNoDelay(true)
-  .connectionPool()
-      .numTestsPerEvictionRun(3)
-      .testOnBorrow(false)
-      .testOnReturn(false)
-      .testWhileIdle(true)
-  .addServer()
-      .host("localhost")
-      .port(11222);
-RemoteCacheManager rmc = new RemoteCacheManager(cb.build());
-
-----
-
-For a complete reference to the available configuration option please refer to the
-link:{javadocroot}/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.html[ConfigurationBuilder]'s javadoc.
-
-It is also possible to configure the Java Hot Rod client using a properties file, e.g.:
-
-[source]
-----
-infinispan.client.hotrod.transport_factory = org.infinispan.client.hotrod.impl.transport.tcp.TcpTransportFactory
-infinispan.client.hotrod.server_list = 127.0.0.1:11222
-infinispan.client.hotrod.marshaller = org.infinispan.jboss.marshalling.commons.GenericJBossMarshaller
-infinispan.client.hotrod.async_executor_factory = org.infinispan.client.hotrod.impl.async.DefaultAsyncExecutorFactory
-infinispan.client.hotrod.default_executor_factory.pool_size = 1
-infinispan.client.hotrod.default_executor_factory.queue_size = 10000
-infinispan.client.hotrod.tcp_no_delay = true
-infinispan.client.hotrod.request_balancing_strategy = org.infinispan.client.hotrod.impl.transport.tcp.RoundRobinBalancingStrategy
-infinispan.client.hotrod.key_size_estimate = 64
-infinispan.client.hotrod.value_size_estimate = 512
-infinispan.client.hotrod.force_return_values = false
-infinispan.client.hotrod.client_intelligence = HASH_DISTRIBUTION_AWARE
-infinispan.client.hotrod.batch_Size = 10000
-
-## below is connection pooling config
-maxActive=-1
-maxTotal = -1
-maxIdle = -1
-whenExhaustedAction = 1
-timeBetweenEvictionRunsMillis=120000
-minEvictableIdleTimeMillis=300000
-testWhileIdle = true
-minIdle = 1
-
-----
-
-The properties file is then passed to one of constructors of link:{javadocroot}/org/infinispan/client/hotrod/RemoteCacheManager.html#RemoteCacheManager-java.net.URL-[RemoteCacheManager].
-You can use property substitution to replace values at runtime with link:https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html[Java system properties]:
-
-[source]
-----
-infinispan.client.hotrod.server_list = ${server_list}
-----
-
-In the above example the value of the _infinispan.client.hotrod.server_list_ property will be expanded to the value of the _server_list_ Java system property.
-
-which means that the value should be taken from a system property named jboss.bind.address.management and if it is not defined use 127.0.0.1.
-
-For a complete reference of the available configuration options for the properties file please refer to link:{javadocroot}/org/infinispan/client/hotrod/configuration/package-summary.html#package.description[remote client configuration] javadoc.
+== Hot Rod Java clients
+Use Java clients to access data through a {brandname} Hot Rod server.
+
+{brandname} provides a reference Hot Rod Java client implementation as part of
```

Comment:
"as part of each distribution" redudant? I'm also not sure what you mean by "each distribution" :slightly_smiling_face:

Reply:
@ryanemerson nice one.